plugins/ssh/komand_ssh/actions/run/action.py

```diff
@@ -14,7 +14,7 @@ def run(self, params={}):
         results = {}
         command = params.get(Input.COMMAND)
         client = self.connection.client(params.get(Input.HOST))
-        (stdin, stdout, stderr) = client.exec_command(command)
+        (_, stdout, stderr) = client.exec_command(command)
```

Comment:
is this an intentional change?

Reply:
Yep, linter was failing because the variable wasn't being used so we can replace it with a _ to say we still want to get it but not use it 🙂